changelog/dev-8113-bump-qit-cli-dep-version

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Bump qit-cli dependency to version 0.4.0.
```

Comment:
Can add a `Comment: ` prefix here to avoid including it in the changelog as it might not be relevant to folks outside A8C.

Reply:
Thanks for the review, Deepak! I wasn't aware of the `Comment: ` prefix feature. I've updated the changelog entry in https://github.com/Automattic/woocommerce-payments/pull/8114/commits/2f65ebe0728bd30f6da7edb68b295679f69339f4.